src/weekly/export.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import csv from 'fast-csv'
+import moment from 'moment'
+import { upload, createReadStream } from './../lib/storage'
+import { ReadStream } from 'fs'
+import {getStage} from './../lib/config'
+import {weeklyOutputFileName as generateOutputFileName} from './../lib/filenames'
+import WeeklyExporter from './WeeklyExporter'
+import type {result, input} from '../exporter'
+
+const SUBSCRIPTION_NAME = 'Subscription.Name'
+const HOLIDAYS_QUERY_NAME = 'WeeklyHolidaySuspensions'
+const SUBSCRIPTIONS_QUERY_NAME = 'WeeklySubscriptions'
+
+function getDownloadStream (results: Array<result>, stage: string, queryName: string): Promise<ReadStream> {
```

Comment:
can't this code (and possibly some other functions here) be shared between home delivery and weekly?